examples/_includes/tab.xml.njk

```diff
@@ -0,0 +1,40 @@
+{% extends 'base.xml.njk' %}
+
+{% block container %}
+  <text style="description">This is the content for tab 1.</text>
```

Comment:
Do we need this?

Reply:
I initially added this to show-case support for content. Moving this from here to each individual tab.